File: laramie-seguridad-webapi/src/domain/entities/acceso.ts
export default class Acceso {

    id: number;
	idUsuario: number;
	idTipoAcceso: number;
	identificador: string;
	password: string;

	constructor(
        id: number = 0,
		idUsuario: number = 0,
		idTipoAcceso: number = 0,
		identificador: string = "",
		password: string = ""
	)
	{
        this.id = id;
		this.idUsuario = idUsuario;
		this.idTipoAcceso = idTipoAcceso;
		this.identificador = identificador;
		this.password = password;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.idUsuario = row.idUsuario ?? 0;
		this.idTipoAcceso = row.idTipoAcceso ?? 0;
		this.identificador = row.identificador ?? "";
		this.password = row.password ?? "";
	}

}

File: modules/seguridad/models/acceso.py
from sqlalchemy import Column, BigInteger, String, ForeignKey
from sqlalchemy.orm import relationship

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Acceso(Base):
    __tablename__ = "seguridad_accesos"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    id_usuario = Column(BigInteger, ForeignKey("seguridad_usuarios.id"), nullable=False)
    id_tipo_acceso = Column(BigInteger, nullable=False, default=10)
    identificador = Column(String(250), nullable=False, index=True)
    password = Column(String(250), nullable=False)

    usuario = relationship("Usuario", back_populates="accesos")
